### ISSUE TITLE
It seemed tiny differences before and after correction using my own retrained model

### ISSUE THREAD

**Issue by hermosayhl** [mahmoudnafifi/Exposure_Correction#17]
I trained the model with the original paired images of ExpertC in MIT-Adobe-5K, only underexposed and well-exposed images are provided.
When I run the demo, I got almost the same image as input, what happend ?

Left is the input and the right one is corresponding output using model_512.mat

![image](https://user-images.githubusercontent.com/49510831/141387858-64831c10-9704-4740-8577-0810aac06348.png)

**Comment by mahmoudnafifi** [comment]
That is odd! Are you sure that the code uses the proper ground-truth directory? Did you try the provided pre-trained model on the same image?

**Comment by hermosayhl** [comment]
Thanks for your reply! 

Listed below are infered results of this picture using your provided pretrained model and it works well

![image](https://user-images.githubusercontent.com/49510831/141426632-b2d963c9-4540-4ea3-b1db-0d6d5b97e580.png)

In the course of training from scratch,  only underexposed and well-exposed images are provided, which is not same as yours. In fact, G_loss, one of the training loss, is decreased to about 0.02 for training and validation.

I'll check more and figure out where I make mistakes.

**Comment by hermosayhl** [comment]
> That is odd! Are you sure that the code uses the proper ground-truth directory? Did you try the provided pre-trained model on the same image?

I checked this. I truly use the proper ground-truth directory. Perhaps, I have changed the code some where. I noticed that those images you used for training are all with a max length of 1024px, which differs from my 512px.
